templates/api/minimal_registration.html

```diff
@@ -73,43 +73,57 @@
         }
 
     </style>
+
+    <script type="text/javascript">
+        function setPasswordType() {
+            // Set type to password so that browser offers to store it
```

Comment:
It would be nice to share this with the main login page (even though we're doing the interface change, etc)